package.json

```diff
@@ -7,13 +7,21 @@
         "build": "next build",
         "start": "next start",
         "export": "next build && next export",
-        "lint": "next lint",
-        "format": "prettier --write \"**/**/*.{js,ts,tsx}\"",
+        "lint": "eslint --cache",
+        "lint:fix": "eslint --cache --fix",
         "test": "jest",
-        "db-restart": "npx supabase stop --no-backup && npx supabase start",
-        "gen-db-types": "npx supabase gen types --lang typescript --local > supabase/database.types.ts",
-        "gen-db-migration": "npx supabase db diff", 
-        "gen-seed-data": "npx --yes tsx supabase/seed.ts > supabase/seed.sql"
+        "setup-git-hooks": "git config --local core.hooksPath .githooks/",
+        "gen-seed-data": "npx tsx supabase/seed.ts > supabase/seed.sql",
+        "postinstall": "husky install && yarn husky-prep-1 && yarn husky-prep-2",
+        "husky-prep-1":"sudo chmod ug+x .husky/*",
+        "husky-prep-2":"sudo chmod ug+x .git/hooks/*",
```

Comment:
I think this sudo is causing the issue

Reply:
yeah, it wouldn't work without sudo in my system, i'll see if it works on vercel without it, but im not sure what's the correct approach, it either works on dev machine (devcontainer) or vercel can't have both?
alternatively we could get rid of node16 from our CI that'd solve all problems because it doesn't need these extra permissions in later version of husky (which is incompatible with node16)